cirq/_doc.py

```diff
@@ -44,4 +46,20 @@ def document(value: Any, doc_string: Optional[str] = None):
     """
     docs = DocProperties(doc_string=doc_string)
     RECORDED_CONST_DOCS[id(value)] = docs
+
+    ## this is how the devsite API generator picks up
+    ## docstrings for type aliases
+    if sys.version_info >= (3, 7):
```

Comment:
why the version check?

Reply:
Copy paste for the TF codebase - before 3.7 type aliases wouldn't work with this. But as @mpharrigan pointed out actually using this _and_ the exception handling is superfluous.